nav_msgs/Path pathToGoal
---
std_msgs/Bool success
